src/test/java/nl/jqno/equalsverifier/integration/extended_contract/SignificantFieldsTest.java

```diff
@@ -163,6 +163,31 @@ public void succeed_whenTwoFieldsAreUnused_givenAllFieldsShouldBeUsedExceptThose
     }
 
     @Test
+    public void succeed_repeating_withIgnoredFields_givenAllFieldsShouldBeUsedExceptThoseTwo() {
```

Comment:
The integration tests have a convention, where the name of each test case is a sort of reverse Given/When/Then. The clauses are separated by `_` for readability, but there should be no `_`s in other spots.